virt/virtual_machines/virt-edit-vms.adoc

```diff
@@ -4,9 +4,7 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-edit-vms
 toc::[]
 
-You can update a virtual machine configuration using either the YAML editor in the
-web console or the OpenShift client on the command line. You can also update
-a subset of the parameters in the *Virtual Machine Overview* of the web console.
+You can update a virtual machine configuration using either the YAML editor in the web console or the OpenShift client on the command line. You can also update a subset of the parameters in the *Virtual Machine Details* screen that displays in the web console.
```

Comment:
s/OpenShift client on the command line/`oc` command line interface

Suggestion to remove "that displays in the web console" in the second sentence.

Reply:
Removed.